install/kubernetes/istio-rbac-alpha.yaml

```diff
@@ -1,14 +1,109 @@
 # Permissions and roles for istio
+# To debug: start the cluster with -vmodule=rbac,3 to enable verbose logging on RBAC DENY
+# Also helps to enable logging on apiserver 'wrap' to see the URLs.
+# Each RBAC deny needs to be mapped into a rule for the role.
+# If using minikube, start with '--extra-config=apiserver.Authorization.Mode=RBAC'
+#
+# NOTE: If deploying istio to a namespace other than 'default' then change the
+# ClusterRoleBinding namspace target appropriately.
```

Comment:
So, keep in mind that @dcberg tweaked the existing RBAC beta to work on RBAC v1alpha1. This is what he got working. So, before we suggest alternatives, we need to make sure that this would work on v1alpha1. I remember trying the RoleBinding and it didn't work (IIRC).